docs/adoc/config-site.adoc

```diff
@@ -0,0 +1,278 @@
+  Allows the values `0` (warnings only), `1` (warnings and informational messages), or `2` (all messages including debugging messages).
+
+`repo_dir`::
+  Specifies the directory that is used to permanently cache remote Git repositories.
+  Git repositories in this directory persist across builds and instances of the {ds2} process.
+
+`temp_repo_dir`::
+  Specifies the directory that is used to temporarily cache specific Git branches.
+  When receiving a build instruction, {ds2} will clone an individual branch of a Git repository from `repo_dir`.
+  It will then build using the data from `temp_repo_dir`.
+  This prevents branch checkout collisions during builds and other problems.
+  After the build instruction has run through,
+
+`max_threads`::
+  Specifies the maximum number of threads to use for {ds2}.
+  You can specify either a number, such as `4` or use `max` to use as many threads as there are logical CPU cores.
++
+In any case, the upper limit to the number of threads is the number of logical CPU cores.
+
+`valid_languages`::
+  Specifies the overall list of language codes recognized as valid by this instance of {ds2}.
+  Language codes must follow the schema `la-ng` (where `la` is the language and `ng` is the regional variety).
+  Languages codes must be specified separated by spaces.
++
+[NOTE]
+.Independent sets of language code definitions
+====
+{ds2} uses two independent sets of language code definitions:
+
+* A set of language codes valid for documents and navigational pages (this attribute, global)
+* A set of language codes that navigational pages are created for (`languages` attribute, target-specific)
+====
+
+
+[#config-site-target-section]
+## Target sections
+
+{ds2} supports reuse of the same `*docserv*` process to synchronize to multiple target servers.
+Each of those target servers is defined in its own target section.
+All or a subset of target servers can share certain parts of their configuration by sharing the specific configuration directories, if desired.
+
+[source,ini]
+----
+[target_0]
+name = internal
+active = yes
+internal = yes
+
+enable_target_sync = yes
+target_path = ssh://user@server:/srv/www/htdocs
+backup_path = /home/docserv/target-internal/
+
+config_dir = /etc/docserv/config.d/
+template_dir = /etc/docserv/templates/
+server_root_files = /etc/docserv/server-root-files-internal
+default_xslt_params = /etc/docserv/xslt-params.txt
+
+enable_ssi_fragments = yes
+fragment_dir = /etc/docserv/fragment-internal
+fragment_l10n_dir = /etc/docserv/fragment-l10n-internal
+
+server_base_path = /
+canonical_url_domain = https://www.example.org
+
+draft = yes
+remarks = yes
+meta = no
+
+languages = en-us
+default_lang = en-us
+omit_default_lang_path = no
+
+site_sections = section-a section-b
+default_site_section = section-a
+
+zip_formats = pdf epub single-html
+----
+
+`[target+++_+++__n__]`::
+  The section identifier which needs to start with `target_`, followed by an integer.
+
+`name`::
```

Comment:
Not sure if it make sense, but having the allowed datatype could be helpful.

Reply:
I guess the issue is that we'd need to properly define data types then. :)

Reply:
I wouldn't expect a rigid or strict datatype system. I would be happy to have some basic types like string, bool, integer etc. Maybe even a URL type to distinguish from string. Just to get an idea what is expected.

Reply:
I have added such information, resolving this too.